impeller/geometry/matrix.h

```diff
@@ -407,6 +407,19 @@ struct Matrix {
 
   std::optional<MatrixDecomposition> Decompose() const;
 
+  bool Equals(const Matrix& matrix, Scalar epsilon = 1e-5f) const {
```

Comment:
We have an existing scalar nearly equal method you could use instead?

Reply:
Done, I also expanded `ScalarNearlyEquals` to use `fabsf` since it will be faster